src/components/section/index.tsx

```diff
@@ -30,7 +30,11 @@ const Section: FC<Props> = ({
 
   return (
     <Stack direction={{ '2xs': 'column', md: 'row' }} spacing="xl">
-      {image ? <Box maxW={maxImgW}>{image}</Box> : null}
+      {image ? (
+        <Box maxW={maxImgW} flexShrink={0}>
```

Comment:
Maybe you can test the pre-release before you merge it. (I know you already did, just saying 🐥)

🌈 thank you!

Reply:
I did :)